transport/src/test/java/io/netty/channel/PendingWriteQueueTest.java

```diff
@@ -248,9 +248,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
     @Test
     public void testCloseChannelOnCreation() {
         EmbeddedChannel channel = new EmbeddedChannel(new ChannelInboundHandlerAdapter());
+        ChannelHandlerContext context = channel.pipeline().firstContext();
```

Comment:
were you intending on using this some where else? Or does the close operation now prevent you from getting the context?